src/SourceBuild/tarball/content/src/source-mappings.json

```diff
@@ -0,0 +1,152 @@
+{
+    "defaults": {
+        "defaultRef": "main",
+        "exclude": [
+            "**/*.dll",
+            "**/*.Dll",
+            "**/*.exe",
+            "**/*.pdb",
+            "**/*.mdb",
+            "**/*.zip",
+            "**/*.nupkg"
+        ]
+    },
+    "mappings": [
+        {
+            "name": "arcade",
+            "defaultRemote": "https://github.com/dotnet/arcade",
+            "include": [
+                "*.*",
+                ".*",
+                "eng/**/*",
+                "src/**/*",
+                "scripts/**/*"
+            ],
+            "exclude": [
+                "**/tests/**/*",
+                "**/*.Tests/**/*"
+            ]
+        },
+        {
+            "name": "aspnetcore",
+            "defaultRemote": "https://github.com/dotnet/aspnetcore",
+            "exclude": [
+                "**/samples/**/jquery-validation-unobtrusive/.bower.json",
```

Comment:
The underlying issue was resolved (https://github.com/dotnet/aspnetcore/pull/43196) so these are no longer necessary.